asciidoc/client-applications.adoc

```diff
@@ -133,6 +133,64 @@ For details on how to configure multi-data center routing policies for a Causal
 ====
 
 
+=== Cluster bootstrap address resolution
+
+The 1.7 series drivers introduced the ability to intercept bootstrap address resolution.
```

Comment:
We try to avoid talking about versions inside the docs, with the assumption "if you are reading about it in the docs that support your version, it is available in that doc". Could this sentence be changed to something similar to:

_Drivers have the ability to intercept bootstrap address resolution._